modules/SiteHeader/assets/scripts/has-sticky-header.js

```diff
@@ -0,0 +1,8 @@
+function registerHasStickyHeaderActions() {
+    window.sitchco.hooks.addAction(window.sitchco.constants.SCROLL, () => {
+        const isSticky = window.scrollY > 10;
```

Comment:
Shouldn't isSticky be triggered based on the dynamically calculated header height in UIFramework? 10 seems like an arbitrary number.